### PR TITLE
Added profile dropdown in login

### LINKED ISSUE #33: Add change password link

### PR DESCRIPTION
**Pull request by ParisPianist196**
Added a profile dropdown in nav
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,5 +9,6 @@
     <body>
         <div id="root"></div>
         <script type="module" src="/src/main.tsx"></script>
+        <script src="node_modules/flowbite/dist/flowbite.min.js"></script>
```
Comment (jwu910) [inline index.html]: we shouldn't need this here btw, we shuold be able to import flowbite into the source code where ever we are using it

**Comment by etudie** [comment]
Doesn't work on the netlify app :(

File: src/components/Navigation/ProfileMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+export default function ProfileMenu() {
+    const signOut = async () => {
+        const { error } = await client.auth.signOut()
+        if (error) throw error
+    }
+    return (
+        <div>
+            <button
+                id="dropdownDefaultButton"
```
Comment (jwu910) [inline src/components/Navigation/ProfileMenu.tsx]: https://www.flowbite-react.com/docs/components/dropdown#default-dropdown

we can look at importing the react component for the dropdown to add in the functionality we're seeking here. I am thinking we were importing the flowbyte lib like a cdn in index.html to try to apply that logic here

**Review by jwu910 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ParisPianist196) [reply index.html]: Didn't know about the react components lol. This makes it way easier!

File: src/components/Navigation/ProfileMenu.tsx (hunk 2 above)
Comment (ParisPianist196) [reply src/components/Navigation/ProfileMenu.tsx]: See above lol

**Comment by ParisPianist196** [comment]
> Doesn't work on the netlify app :(

Turns out the js wasn't loading. Resolved using the react component.

**Review by jwu910 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (jwu910) [reply index.html]: Hehe all about that write less code life

**Pull request merged**